projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,43 @@
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define TIMER_PERIOD 500  // timer in ms
```

Comment:
Our usual convention is to suffix variables with the unit they're in, so this should be `TIMER_PERIOD_MS`. Great job using a #define though, most people don't think to!

Reply:
Fixed tnx